skfda/exploratory/visualization/_parametric_plot.py

```diff
@@ -0,0 +1,103 @@
+"""Phase-Plane Plot Module.
+
+This module contains the functionality in charge of plotting
+two different functions as coordinates, this can be done giving
+one FData, with domain 1 and codomain 2, or giving two FData, both
+of them with domain 1 and codomain 1.
+"""
+
+from typing import Any, Optional, Union
+
+from matplotlib.axes import Axes
+from matplotlib.figure import Figure
+
+from ...representation import FData
+from ._utils import _get_figure_and_axes, _set_figure_layout
+
+
+class ParametricPlot:
+    """
+    Parametric Plot visualization.
+
+    This class contains the functionality in charge of plotting
+    two different functions as coordinates, this can be done giving
+    one FData, with domain 1 and codomain 2, or giving two FData, both
+    of them with domain 1 and codomain 1.
+    Args:
+        fdata1: functional data set that we will use for the graph. If it has
+            a dim_codomain = 1, the fdata2 will be needed.
+        fdata2: optional functional data set, that will be needed if the fdata1
+            has dim_codomain = 1.
+    """
+
+    def __init__(
+        self,
+        fdata1: FData,
+        fdata2: Optional[FData] = None,
+    ) -> None:
+        self.fdata1 = fdata1
+        self.fdata2 = fdata2
+
+    def plot(
+        self,
+        chart: Union[Figure, Axes, None] = None,
+        *,
+        fig: Optional[Figure] = None,
+        ax: Optional[Axes] = None,
+        **kwargs: Any,
+    ) -> Figure:
+        """
+        ParametricPlot graph.
+
+        Plot the functions as coordinates. If two functions are passed
+        it will concatenate both into one only FData.
+        Args:
+            chart: figure over with the graphs are plotted or axis over
+                where the graphs are plotted. If None and ax is also
+                None, the figure is initialized.
+            fig: figure over with the graphs are plotted in case ax is not
+                specified. If None and ax is also None, the figure is
+                initialized.
+            ax: axis where the graphs are plotted. If None, see param fig.
+            kwargs: optional arguments.
+        Returns:
+            fig (figure object): figure object in which the ParametricPlot
+            graph will be plotted.
+        """
+        fig, axes = _get_figure_and_axes(chart, fig, ax)
+
+        if self.fdata2 is not None:
+            self.fd_final = self.fdata2.concatenate(
+                self.fdata1, as_coordinates=True,
+            )
+            x_label = self.fd_final.coordinate_names[1]
```

Comment:
Why here and why in inverse order?

Reply:
when the concatenate is done, i have realised that the coordinate names are inverted, so i have to do this to avoid the coordinates being put wrongly

Reply:
Well, that is because you are concatenating the functions in reverse instead of in the right order. Try
```python
skfda.concatenate((self.fdata1, self.fdata2), as_coordinates=True)
```
instead.

Reply:
no, because the thing was because i did it before that way. The problem is if i concatenate it as you say it will go to x-axis the function that should go to the y-axis

Reply:
Then it is a bug in the concatenate function. Please report the behaviour you saw in an issue and we will fix this before merging this PR.

Reply:
ok i will report it and correct it so it works with that change

Reply:
my bad, it actually worked but i had other problem, solved